src/renderer/components/_atoms/buttons/variables-table/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { cn } from '@root/utils'
+import { ComponentPropsWithoutRef, ReactNode } from 'react'
+
```

Comment:
When using 'ComponentPropsWithoutRef', you don't need to put 'className' and 'children' as additional props, once they are default properties in every React component.